src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.FruitStrategy;
+import core.basesyntax.strategy.OperationsStrategy;
+import core.basesyntax.strategy.impl.BalanceOperation;
+import core.basesyntax.strategy.impl.FruitStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseOperation;
+import core.basesyntax.strategy.impl.ReturnOperation;
+import core.basesyntax.strategy.impl.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FruitServiceImplTest {
+    private static FruitService fruitService;
+    private static FruitStrategy fruitStrategy;
+
+    @BeforeAll
+    static void beforeAll() {
+        Map<FruitTransaction.Operation, OperationsStrategy> operationsStrategyMap = new HashMap<>();
+        operationsStrategyMap.put(FruitTransaction.Operation.BALANCE, new BalanceOperation());
+        operationsStrategyMap.put(FruitTransaction.Operation.RETURN, new ReturnOperation());
+        operationsStrategyMap.put(FruitTransaction.Operation.PURCHASE, new PurchaseOperation());
+        operationsStrategyMap.put(FruitTransaction.Operation.SUPPLY, new SupplyOperation());
```

Comment:
You can use static import insted FruitTransaction.Operation... use just PURCHASE, and I think that method Map.of() better to readibility